include/alpaka/kernel/TaskKernelGpuCudaRt.hpp

```diff
@@ -497,9 +497,9 @@ namespace alpaka
                     // This forces the type of a float argument given with std::forward to this function to be of type float instead of e.g. "float const & __ptr64" (MSVC).
                     // If not given by value, the kernel launch code does not copy the value but the pointer to the value location.
                     meta::apply(
-                        [&](TArgs ... args)
+                        [&](typename std::decay<TArgs>::type const & ... args)
```

Comment:
This conflicts with the comment now. For HIP you have removed the comment. Do you want to remove it here a well?

Reply:
Yes must this comment should be removed too